inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleService.java

```diff
@@ -15,20 +15,24 @@
  * limitations under the License.
  */
 
-package org.apache.inlong.manager.service.core;
+package org.apache.inlong.manager.service.user;
 
-import java.util.List;
+import org.apache.inlong.manager.pojo.user.TenantRoleInfo;
+import org.apache.inlong.manager.pojo.user.TenantRolePageRequest;
+import org.apache.inlong.manager.pojo.user.TenantRoleRequest;
+
+import com.github.pagehelper.PageInfo;
 
 /**
  * Role service
  */
-public interface RoleService {
-
-    /**
-     * Get roles based on username
-     *
-     * @param username username
-     * @return Role list
-     */
-    List<String> listByUser(String username);
+public interface TenantRoleService {
+
+    PageInfo<TenantRoleInfo> listByCondition(TenantRolePageRequest request);
```

Comment:
Please add Javadoc for those methods in interface class.

Reply:
fixed, thx